OlatunjiYso/NairobiJavaDev
# #156037248: Revamp UI and enable swipe-down reload

#### What does this PR do?
This PR makes the developers display in grid of two columns
It adds user feedback when making requests

#### Description of Task to be completed?
- Make developers card appear in a grid system.
-  Add a spinner that rolls when request is yet to be completed
- Add swipe down to update feature
#### How should this be manually tested?
- pull this branch 
- Setup on android studio
- Run on emulator
- Swipe down and see it relaod as described

#### What are the relevant pivotal tracker stories?
[Story - 156037248](https://www.pivotaltracker.com/n/projects/2155295)
#### Screenshots (if appropriate)
![screenshot_20180920-011811](https://user-images.githubusercontent.com/27797745/45789231-7f239000-bc76-11e8-8597-ded3e43a62d6.png)
![screenshot_20180920-012256](https://user-images.githubusercontent.com/27797745/45789232-7f239000-bc76-11e8-89b3-75d3f1298bd9.png)

#### Questions:
n/a
